src/lib.rs

```diff
@@ -2079,7 +2085,27 @@ impl RtcConfig {
     ///
     /// This includes setting the initial estimate to start with.
     pub fn enable_bwe(mut self, initial_estimate: Option<Bitrate>) -> Self {
-        self.bwe_initial_bitrate = initial_estimate;
+        match initial_estimate {
+            Some(b) => {
+                let conf = self.bwe_config.get_or_insert(BweConfig::new(b));
+                conf.initial_bitrate = b;
+            }
+            None => {
+                self.bwe_config = None;
+            }
+        }
+
+        self
+    }
+
+    /// Enable the experimental loss based BWE subsystem.
+    pub fn enable_bwe_loss_experiment(mut self, enabled: bool) -> Self {
```

Comment:
Maybe this should just be called `enable_bwe_loss` and we can not in the comment that it's experimental. Then, when it becomes stable, we can default it to on

Reply:
Yeah, let's just make it the way it will be.